app/src/androidTest/kotlin/com/bitchat/EncryptionServiceTest.kt

```diff
@@ -0,0 +1,30 @@
+package com.bitchat
+
+import android.content.Context
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import com.bitchat.android.crypto.EncryptionService
+import com.bitchat.android.crypto.EncryptionService.Companion.PUBLIC_KEY_DATA_SIZE
+import junit.framework.TestCase.assertEquals
+import junit.framework.TestCase.assertTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
```

Comment:
I'm not sure why we need an instrumented test for this case. 

You can just mock or fake the context and write a junit tests 

> [!NOTE]
>Junit tests are much faster than automated UI tests. 
Write automated tests for verifying the behaviour. For logics and all  junit tests that doesn't involve a device is great.

Reply:
I was looking for the context, I think mocking is a good idea. I will refactor it to unit tests